test/integration/admin/api/account/authentication_providers_controller_test.rb

```diff
@@ -31,9 +32,10 @@ def setup
   end
 
   test '#create creates 2 times with the same data' do
+    params = authentication_provider_params
     assert_difference @provider.self_authentication_providers.method(:count), 2 do
       2.times do
-        post admin_api_account_authentication_providers_path(authentication_provider_params)
+        post admin_api_account_authentication_providers_path(params)
```

Comment:
We don't need to extract `authentication_provider_params` to a var, right? it's only being used once in this test

Reply:
Well, but it is inside `2.times`, so extracting the value to a local var prevents the unnecessary "calculation".

Reply:
Ah, right